pkg/cassandra/config/config.go

```diff
@@ -128,7 +129,16 @@ func (c *Configuration) NewCluster() *gocql.ClusterConfig {
 	if c.Port != 0 {
 		cluster.Port = c.Port
 	}
-	cluster.Compressor = gocql.SnappyCompressor{}
+
+	fmt.Println("Disable Compression Flag is: ")
+	fmt.Println(c.DisableCompression)
+	if !c.DisableCompression {
+		cluster.Compressor = gocql.SnappyCompressor{}
+	}
+
+	fmt.Println("Compression is:")
+	fmt.Println(cluster.Compressor)
```

Comment:
please remove print statements, we don't do that in Jaeger. I believe at some point we simply log the whole config struct, using the logger, not println.

Reply:
My bad..Removed these in the latest push